resources/views/individuals/edit/experiences.blade.php

```diff
@@ -23,21 +24,23 @@
                             ' ' .
                             __('(optional)')" :shortLabel="__('lived experiences')" :hint="__(
                             'Feel free to self-identify your experiences of disability, if you feel it is relevant to your work.',
-                        )" />
+                        )" :interpretationName="'What are your lived experiences of disability or other intersectional identities? (optional)'" />
```

Comment:
As mentioned yesterday you can write this and similar cases like the following:

```suggestion
                        )" interpretationName="What are your lived experiences of disability or other intersectional identities? (optional)" />
```

The ":" at the front of the attribute name basically tells it to interpret it as a php; however since this is just a string value you can just pass in the string directly as a normal attribute.